cpp/CMakeLists.txt

```diff
@@ -261,14 +287,13 @@ endif(CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES)
 include_directories("${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/src"
-                    "${CMAKE_SOURCE_DIR}/../thirdparty/cub"
-                    "${CMAKE_SOURCE_DIR}/../thirdparty/libcudacxx/include"
-                    "${GDAL_INCLUDE_DIRS}"
+                    "${GDAL_INCLUDE}"
                     "${RMM_INCLUDE}"
                     "${CUDF_SRC_INCLUDE}"
                     "${CUDF_INCLUDE}"
-                    "${CUB_INCLUDE}"
-                    "${CUDACXX_INCLUDE}")
+                    "${CUB_INCLUDE_DIR}"
```

Comment:
All 3 of these are in the CUDA toolkit as of CUDA 11 so need to be moved above the CUDA toolkit include paths (which need to be explicitly added). See libcudf cmake.